crates/forge/tests/cli/test_cmd.rs

```diff
@@ -274,6 +274,55 @@ Ran 1 test suite [ELAPSED]: 1 tests passed, 0 failed, 0 skipped (1 total tests)
 "#]]);
 });
 
+const SIMPLE_CONTRACT: &str = r#"
+import {Test} from "forge-std/Test.sol";
+
+contract SimpleContract {
+    uint256 public num;
+
+    function setValues(uint256 _num) public {
+        num = _num;
+    }
+}
+
+contract SimpleContractTest is Test {
+    function test() public {
+        SimpleContract c = new SimpleContract();
+        c.setValues(100);
+    }
+}
+   "#;
+
+forgetest_init!(can_run_test_with_json_output_verbose, |prj, cmd| {
+    prj.wipe_contracts();
+    prj.clear();
```

Comment:
use `forgetest!` without init if you're not going to use the initial project layout, we should go through existing tests to check for this too